dashboard/apps/prototype/management/commands/helpers.py

```diff
@@ -73,3 +81,71 @@ def print_task(task, start_date, end_date, padding='  '):
         else:
             logger.info('%s  %s', padding, line)
     return time_spent, money_spent
+
```

Comment:
they will go there if we find them useful to be used elsewhere. these functions were originally in `queries.py`, which doesn't exist anymore. before removing, i did some refactoring and then relocated them into two places.

1. more useful ones went into custom Managers in `models.py`
2. those only used by commands went into this `helpers.py`